src/main/java/io/github/org/programming/bot/commands/thread/ActiveQuestionsHandler.java

```diff
@@ -0,0 +1,175 @@
+        String categoryCapitalised =
+                newCategory.substring(0, 1).toUpperCase() + newCategory.substring(1);
+
+
+        for (String line : lines) {
+            if (line.contains(categoryCapitalised)) {
+                newContent.append(line).append("\n").append(format(channelLink)).append("\n");
+            } else {
+                newContent.append(line).append("\n");
+            }
+        }
+
+        message.editMessage(newContent.toString()).queue();
+    }
+
+    private static String format(String channelLink) {
+        // add a bullet point before the link
+        return "• " + channelLink;
+    }
+
+    public static void updateActiveQuestionMessage(String guildId, String messageId) {
+        getContext()
+            .insertInto(ACTIVEQUESTIONMESSAGE, ACTIVEQUESTIONMESSAGE.GUILD_ID,
+                    ACTIVEQUESTIONMESSAGE.MESSAGE_ID)
+            .values(guildId, messageId)
+            .execute();
+    }
+
+    public static @Nullable String getActiveQuestionMessage(String guildId) {
+        return getContext().select(ACTIVEQUESTIONMESSAGE.MESSAGE_ID)
+            .from(ACTIVEQUESTIONMESSAGE)
+            .where(ACTIVEQUESTIONMESSAGE.GUILD_ID.eq(guildId))
+            .fetchOne(ACTIVEQUESTIONMESSAGE.MESSAGE_ID);
+    }
+
+    public static void deleteActiveQuestionMessageId(String guildId, String messageId) {
+        getContext().deleteFrom(ACTIVEQUESTIONMESSAGE)
+            .where(ACTIVEQUESTIONMESSAGE.GUILD_ID.eq(guildId))
+            .and(ACTIVEQUESTIONMESSAGE.MESSAGE_ID.eq(messageId))
+            .execute();
+    }
+
+    public static void setMessage(Message message) {
+        ActiveQuestionsHandler.message = message;
+    }
+
+    public static String messageToSend() {
+        return "***Active questions***:".concat("\n")
+            .concat("\n")
+            .concat("**Java**: ")
+            .concat("\n")
+            .concat("\n")
+            .concat("**C++**: ")
+            .concat("\n")
+            .concat("\n")
+            .concat("**C#**: ")
+            .concat("\n")
+            .concat("\n")
+            .concat("**C**: ")
+            .concat("\n")
+            .concat("\n")
+            .concat("**Python**: ")
+            .concat("\n")
+            .concat("\n")
+            .concat("**Js**: ")
+            .concat("\n")
+            .concat("\n")
+            .concat("**PHP**: ")
+            .concat("\n")
+            .concat("\n")
+            .concat("**Go**: ")
+            .concat("\n")
+            .concat("\n")
+            .concat("**Rust**: ")
+            .concat("\n")
+            .concat("\n")
+            .concat("**Swift**: ")
+            .concat("\n")
+            .concat("\n")
+            .concat("**Kotlin**: ")
+            .concat("\n")
+            .concat("\n")
+            .concat("**Ts**: ")
+            .concat("\n")
+            .concat("\n")
+            .concat("**Other**: ")
+            .concat("\n")
+            .concat("\n");
+    }
```

Comment:
we could perhaps factorize the code
and store the languages/frameworks in a single place (an array for example?)

Reply:
Done with commit b63fdb3d7d697582b4d85c4f335107fb82b6a743